### PR TITLE
DOCS-1717-3.17.2

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 1]
```diff
@@ -1,4 +1,271 @@
 [
+  {
+    "title": "v3.17.2",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.30.5",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: Operator version is v1.30.6

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 2]
```diff
@@ -1,4 +1,271 @@
...
+      "eck-kibana": {
+        "version": "7.17.11"
+      },
+      "kibana": {
+        "image": "tigera/kibana",
+        "version": "v3.17.2"
+      },
+      "eck-elasticsearch": {
+        "version": "7.17.11"
+      },
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: `eck-elasticsearch` and `eck-kibana` are now at "7.17.13".

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 3]
```diff
@@ -1,4 +1,271 @@
...
+      "eck-elasticsearch-operator": {
+        "version": "2.6.1"
+      },
+      "elasticsearch-operator": {
+        "image": "tigera/eck-operator",
+        "version": "v3.17.2"
+      },
+      "coreos-alertmanager": {
+        "version": "v0.25.0"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json:226]: v0.25.1

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 4]
```diff
@@ -1,4 +1,271 @@
...
+        "image": "tigera/honeypod-exp-service",
+        "version": "v3.17.2"
+      },
+      "honeypod-controller": {
+        "image": "tigera/honeypod-controller",
+        "version": "v3.17.2"
+      },
+      "key-cert-provisioner": {
+        "image": "tigera/key-cert-provisioner",
+        "version": "v1.1.10",
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: v1.1.13

**Review by danudey — CHANGES_REQUESTED** [review]
Quite a few smaller version changes this time around, all in comments.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json:226]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 5]
```diff
@@ -1,4 +1,271 @@
...
+      "packetcapture": {
+        "image": "tigera/packetcapture",
+        "version": "v3.17.2"
+      },
+      "prometheus": {
+        "image": "tigera/prometheus",
+        "version": "v3.17.2"
+      },
+      "coreos-prometheus": {
+        "version": "v2.43.1"
```
Comment (danudey) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: v2.47.0

**Review by danudey — CHANGES_REQUESTED** [review]

**Review by danudey — APPROVED** [review]
lgtm, thanks!

File: calico-enterprise_versioned_docs/version-3.17/releases.json (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.17/releases.json [hunk 6]
```diff
@@ -1,4 +1,271 @@
...
+      },
+      "eck-elasticsearch-operator": {
+        "version": "2.6.1"
+      },
+      "elasticsearch-operator": {
+        "image": "tigera/eck-operator",
+        "version": "v3.17.2"
+      },
+      "coreos-alertmanager": {
+        "version": "v0.25.1"
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.17/releases.json]: I noticed today that this should actually be v0.25.0

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/releases.json]: Fixed.

**Pull request merged**